package/src/wallmon.cpp

```diff
@@ -30,7 +30,7 @@ std::pair<int, unsigned long long> wallmon::get_mother_starttime(
   std::vector<std::string> stat_entries{};
   stat_entries.reserve(52);
   std::string tmp_str{};
-  unsigned long long start_time_clock_t;
+  unsigned long long start_time_clock;
```

Comment:
I think this is an uncecessary change

Reply:
That's indeed not necessary. I just renamed the variable as typically typedefs get an `_t` suffix but that's purely stylistic. Let me revert this.